api/v1alpha1/pulp_types.go

```diff
@@ -277,6 +278,11 @@ type Api struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:io.kubernetes:Probe","urn:alm:descriptor:com.tectonic.ui:advanced"}
 	LivenessProbe *corev1.Probe `json:"livenessProbe,omitempty"`
+
+	// PodDisruptionBudget is an object to define the max disruption that can be caused to a collection of pods
+	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:hidden"}
```

Comment:
why not advanced instead of hidden?

Reply:
I'm still not 100% sure if we should configure this through the operator.
Maybe users should configure it by themselves because, even though it is an "important" feature to avoid cluster disruption, it is not very common to see users configuring it in the right way.